### ISSUE TITLE
cimi: $first and $last doesn't work after ~ 200000

### ISSUE THREAD

**Issue by schaubl** [slipstream/SlipStreamServer#1630]
How to reproduce:
- https://nuv.la/api/metering?$first=199999&$last=200000 works
- https://nuv.la/api/metering?$first=200000&$last=200000 works
- https://nuv.la/api/metering?$first=200000&$last=200001 doesn't work
- https://nuv.la/api/metering?$first=199999&$last=209000 doesn't work
- https://nuv.la/api/metering?$first=200000 doesn't work
- https://nuv.la/api/metering?$first=199999 doesn't work
- https://nuv.la/api/metering?$first=190002 doesn't work
- https://nuv.la/api/metering?$first=190000 works

Basically if provided/computed $last is above 200000 it doesn't work.

**Comment by loomis** [comment]
The paging parameters from Elasticsearch cannot be used to retrieve documents outside of the "max_result_window" size.  See the [description of the from/size parameters](https://www.elastic.co/guide/en/elasticsearch/reference/current/full-text-queries.html) in the Elasticsearch documentation.  The default "max_result_window" size defaults to 10'000.  Someone has set the value to 200'000 on our Elasticsearch instance, which accounts for the behavior that you see.

Large scale scanning/updating of documents needs to be done directly with the scrolling features of the Elasticsearch API.  The metering service itself does this.
